account-settings/set_user_preferences.py

```diff
@@ -42,30 +57,22 @@ def login(login_url):
 def get_users(i):
     print("Actual page")
     print(i)
-    login(login_url)
     post_url = url+"/api/v4/users"
     payload = {'page': i, 'per_page': '200'}
```

Comment:
Suggest pulling up `200` into a constant to be referenced here and below when computing the total number of pages.

Reply:
True. Right now the API max is 200 per page.